src/User_Command.php

```diff
@@ -290,11 +290,15 @@ public function delete( $args, $assoc_args ) {
 			$users,
 			$assoc_args,
 			function ( $user ) use ( $network, $reassign ) {
+				global $blog_id;
 				$user_id = $user->ID;
 
 				if ( $network ) {
 					$result  = wpmu_delete_user( $user_id );
 					$message = "Deleted user {$user_id}.";
+				} else if (is_multisite() && empty($user->roles)) {
+					$message = "No roles found for user {$user_id} on " . get_site_url($blog_id) . ', no users deleted.';
```

Comment:
Should we use `home_url()` like below?

Also, there will be some PHPCS warnings on your current code.

Reply:
Done!

Reply:
My computer is running php `PHP 8.2.4` and I'm getting a single error that prevents phpcs from working properly.
I had to do `phpcs --standard=WP_CLI_CS  User_Command.php -d error_reporting="E_ALL&~E_DEPRECATED` to actaully get accurate( I hope ) report from running phpcs.